python/ray/air/session.py

```diff
@@ -0,0 +1,233 @@
+import abc
+from enum import Enum, auto
+import logging
+from typing import Optional, Dict
+
+from ray.air.checkpoint import Checkpoint
+
+logger = logging.getLogger(__name__)
+
+
+class SessionType(Enum):
+    TRAIN = auto()
+    TUNE = auto()
```

Comment:
Can we not expose this? I don't think the user should be able to check the difference.